app/views/contexts/summary/group.py

```diff
@@ -128,7 +138,7 @@ def _build_blocks_and_links(
 
         return blocks
 
-    def serialize(self):
+    def serialize(self) -> Mapping[str, Any]:
```

Comment:
Should be concrete return type.

Reply:
👍

Reply:
This function returns:
```
return self.placeholder_renderer.render(
            {
                "id": self.id,
                "title": self.title,
                "blocks": self.blocks,
                "links": self.links,
                "placeholder_text": self.placeholder_text,
            },
            self.location.list_item_id,
        )
```
which I believe is a generic function and it returns Mapping[str, Any]

Correct me if I am wrong.

Reply:
👍